packages/components/releaseNotes/components.md

```diff
@@ -1,9 +1,14 @@
 # @labkey/components
 Components, models, actions, and utility functions for LabKey applications and pages
 
+### version TBD
+*Release*: TBD
+- Issue 52264: Include `viewName` when building query parameters for QueryController selectX actions
+
 ### version 6.31.6
 *Released*: 13 March 2025
 - Issue 52430: Sample Manager: sample names with newline characters
+- Issue 52198: don't show option to validate user lookups
```

Comment:
I assume this was supposed to go up with the new version instead of 6.31.6

Reply:
and is the issue number 52198 or 52189?

Reply:
fixed